PSReleaseTools.psm1

```diff
@@ -147,16 +157,22 @@ Function Save-PSReleaseAsset {
                 else {
                     Throw "Cannot validate path $_"
                 }
-            })]    
+            })]
         [string]$Path = ".",
+
+        [parameter(Mandatory=$false, ParameterSetName="Family")]
+        [parameter(Mandatory=$false, ParameterSetName="All")]
+        [Switch]
+        $Pre,

```

Comment:
Is this the correct location and method for adding the '-pre' switch to the `Save-PSReleaseAsset` function? My testing showed that PowerShell refuses to recognize the switch as a valid parameter for `Save-PSReleaseAsset`, but I cannot figure out why. It might have been that PowerShell was still relying on the old implmentation of `Save-PSReleaseAsset`, however, it was not relying on the old implementations of the other functions in the module (i.e., it loaded all the new functions).